lib/Twig/ExpressionParser.php

```diff
@@ -44,7 +58,11 @@ public function parseExpression($precedence = 0)
             $op = $this->binaryOperators[$token->getValue()];
             $this->parser->getStream()->next();
 
-            if (isset($op['callable'])) {
+            if ('is not' === $token->getValue()) {
+                $expr = $this->parseNotTestExpression($expr);
+            } elseif ('is' === $token->getValue()) {
+                $expr = $this->parseTestExpression($expr);
+            } elseif (isset($op['callable'])) {
```

Comment:
no, I don't think so, that's the only way to support advanced use cases, even if I doubt people are doing this.